.gitlab-ci.yml

```diff
@@ -349,16 +344,22 @@ build:quick:
     when: always
 
 windows64:
-  extends: .windows-template
+  extends: .platform-template
   variables:
     ARCH: "64"
+  script:
+    - call dev/ci/platform-windows.bat
+  tags:
+    - windows-inria
 
 windows32:
-  extends: .windows-template
+  extends: .platform-template
   variables:
     ARCH: "32"
-  except:
-    - /^pr-.*$/
```

Comment:
Is the idea that since we build less we can now build more?

Reply:
I've added ti back (I don't know what it does, not why I did remove it)

Reply:
It excludes the Windows 32 bit job from PR builds because it's usually sufficient to test one architecture.

Reply:
Hum, I'm fine with that, but then I'll remove the entire job.

Reply:
Yes, that makes sense if you're already testing this with a scheduled job on the platform repo.

Reply:
done